src/object/schema_manager.c

```diff
@@ -131,9 +131,86 @@ typedef struct schema_def
 
 static SCHEMA_DEF Current_Schema = { {'\0'}, NULL, NULL, NULL, NULL };
 
+typedef struct system_class_def
+{
+  const char *name;
+  int len;
+} SYSTEM_CLASS_DEF;
 
+// *INDENT-OFF*
+static SYSTEM_CLASS_DEF system_classes[] = {
```

Comment:
If you collect names starting with "_db_" and "db_" and divide them into two arrays, the size is small enough that you do not need to use binary search.
In that case, sequential search seems to be more efficient.
Binary search becomes rather inefficient when the number of data is less than a certain number. This is my personal opinion. Note please

Reply:
Thank you.
However, it becomes difficult to manage the list of system classes.
Instead, I will improve it in other ways.